lib/course_planner_web/controllers/class_controller.ex

```diff
@@ -13,7 +13,8 @@ defmodule CoursePlannerWeb.ClassController do
     join: co in assoc(oc, :course),
     join: c in assoc(oc, :classes),
     preload: [offered_courses: {oc, classes: c, course: co}],
-    order_by: [asc: t.start_date, asc: co.name, asc: c.date, asc: c.starting_at, asc: c.finishes_at]
+    order_by: [desc: t.start_date, desc: co.name, desc: c.date,
+               desc: c.starting_at, desc: c.finishes_at]
```

Comment:
Can you move this query inside `classes.ex`?

Reply:
done